src/auth.js

```diff
@@ -0,0 +1,95 @@
+import auth0 from 'auth0-js'
+import Vue from 'vue'
+
+// exchange the object with your own from the setup step above.
+let webAuth = new auth0.WebAuth({
```

Comment:
Abstract configuration to a `config` object?

Reply:
Yeah, good idea.

Reply:
Kept as TODO for now, but won't be big change.